https://sourceforge.net/p/miktex/bugs/2483/

--- Libraries/3rd/gsoap/source/gsoap/stdsoap2.cpp.orig
+++ Libraries/3rd/gsoap/source/gsoap/stdsoap2.cpp
@@ -5351,7 +5351,7 @@ soap_bind(struct soap *soap, const char *host, int port, int backlog)
   }
 #endif
 #ifdef TCP_FASTOPEN
-  if (!(soap->omode & SOAP_IO_UDP) && setsockopt(soap->master, SOL_TCP, TCP_FASTOPEN, (char*)&set, sizeof(int)))
+  if (!(soap->omode & SOAP_IO_UDP) && setsockopt(soap->master, IPPROTO_TCP, TCP_FASTOPEN, (char*)&set, sizeof(int)))
   { /* silently ignore */
     DBGLOG(TEST, SOAP_MESSAGE(fdebug, "setsockopt TCP_FASTOPEN failed in soap_bind()\n"));
   }
