--- Libraries/MiKTeX/Core/Session/SessionImpl.h.orig
+++ Libraries/MiKTeX/Core/Session/SessionImpl.h
@@ -89,6 +89,17 @@ inline bool operator< (const LanguageInfo_ & lhs, const LanguageInfo_ & rhs)
   }
   return lhs.key < rhs.key;
 }
+static bool operator< (const LanguageInfo_ & lhs, const LanguageInfo_ & rhs) {
+    if (lhs.key == "english")
+    {
+      return true;
+    }
+    if (rhs.key == "english")
+    {
+      return false;
+    }
+    return lhs.key < rhs.key;
+}
 
 struct InternalFileTypeInfo : public MiKTeX::Core::FileTypeInfo
 {
