--- Libraries/MiKTeX/Core/Session/SessionImpl.h.diff
+++ Libraries/MiKTeX/Core/Session/SessionImpl.h
@@ -91,6 +91,17 @@ public:
   }
 
 };
+static bool operator< (const LanguageInfo_ & lhs, const LanguageInfo_ & rhs) {
+    if (lhs.key == "english")
+    {
+      return true;
+    }
+    if (rhs.key == "english")
+    {
+      return false;
+    }
+    return lhs.key < rhs.key;
+}
 
 struct InternalFileTypeInfo : public MiKTeX::Core::FileTypeInfo
 {
