https://sourceforge.net/p/miktex/bugs/2484/
# doesn't work

--- Libraries/MiKTeX/App/CMakeLists.txt.orig
+++ Libraries/MiKTeX/App/CMakeLists.txt
@@ -22,6 +22,7 @@ set(MIKTEX_CURRENT_FOLDER "${MIKTEX_IDE_MIKTEX_LIBRARIES_FOLDER}/app")
 include_directories(BEFORE
   ${CMAKE_CURRENT_SOURCE_DIR}
   ${CMAKE_CURRENT_BINARY_DIR}
+  ${CMAKE_CURRENT_SOURCE_DIR}/../UI/include
 )
 
 set(public_include_directories
