--- cmake/Modules/FindAIDA.cmake.orig
+++ cmake/Modules/FindAIDA.cmake
@@ -30,10 +30,7 @@ else()
     OUTPUT_VARIABLE AIDA_LIBRARIES
     OUTPUT_STRIP_TRAILING_WHITESPACE)
 
-  execute_process(
-    COMMAND ${AIDA_CONFIG_EXECUTABLE} --implementation
-    OUTPUT_VARIABLE AIDA_IMPLEMENTATION
-    OUTPUT_STRIP_TRAILING_WHITESPACE)
+  set(AIDA_IMPLEMENTATION "MacPorts")
 
   # Make variables changeble to the advanced user
   mark_as_advanced(AIDA_CONFIG_EXECUTABLE)
