--- magpy/gui/xmagpy.py.orig
+++ magpy/gui/xmagpy.py
@@ -1,25 +1,7 @@
-import sys
-from socket import gethostname
+#! /usr/bin/env python
 
-try:
-    from magpy.gui.magpy_gui import *
-except:
+from magpy.gui.magpy_gui import *
 
-    planet = gethostname()
-    leonplanets = ['Earth', 'uranus', 'saturn']
-
-    if planet == 'zagll1':
-        magpypath = '/home/rachel/Software/MagPyDev/magpy/trunk/src/'
-    elif planet in leonplanets:
-        magpypath = '/home/leon/Software/magpy/trunk/src/'
-    elif planet == 'Venus':
-        magpypath = '/home/leon/Software/magpy/'
-    else:
-        magpypath = '/home/leon/Software/magpy/trunk/src/'
-
-    sys.path.append(magpypath)
-    from magpy.gui.magpy_gui import *
-  
 app = wx.App(redirect=False)
 frame = MainFrame(None,-1,"")
 frame.Show()
