--- source/interfaces/basic/src/G4UIQt.cc.orig
+++ source/interfaces/basic/src/G4UIQt.cc
@@ -1585,14 +1585,15 @@ void G4UIQt::AddIcon(const char* aLabel, const char* aIconFile, const char* aCom
   QToolBar *currentToolbar = NULL;
   if (userToolBar) {
     if (fToolbarUser == NULL) {
-      fToolbarUser = new QToolBar(fMainWindow);
+      fToolbarUser = new QToolBar(/*fMainWindow*/);
       fToolbarUser->setIconSize (QSize(20,20));
       fMainWindow->addToolBar(Qt::TopToolBarArea, fToolbarUser);
+      fMainWindow->show();
     }
     currentToolbar = fToolbarUser;
   } else {
     if (fToolbarApp == NULL) {
-      fToolbarApp = new QToolBar(fMainWindow);
+      fToolbarApp = new QToolBar(/*fMainWindow*/);
       fToolbarApp->setIconSize (QSize(20,20));
       fMainWindow->addToolBar(Qt::TopToolBarArea, fToolbarApp);
     }
