--- cmake/modules/RootBuildOptions.cmake.orig
+++ cmake/modules/RootBuildOptions.cmake
@@ -180,7 +180,7 @@ if(sysinstall)
   set(BIN_INSTALL_DIR     bin)
   set(LIB_INSTALL_DIR     lib/root)
   set(INCLUDE_INSTALL_DIR include/root)
-  set(ETC_INSTALL_DIR     /etc/root)
+  set(ETC_INSTALL_DIR     etc/root)
   set(MAN_INSTALL_DIR     share/man)
   set(DATA_INSTALL_DIR    share/root)
   set(MACRO_INSTALL_DIR   ${DATA_INSTALL_DIR}/macros)
