diff --git a/CMakeLists.txt b/CMakeLists.txt
index d81cb5b..cc7cca8 100644
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -80,7 +80,7 @@ include(RootConfiguration)
 if(NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_INSTALL_PREFIX)
   install(FILES LICENSE DESTINATION ${CMAKE_INSTALL_DOCDIR})
   install(DIRECTORY README/ DESTINATION ${CMAKE_INSTALL_DOCDIR} PATTERN ".svn" EXCLUDE)
-  install(DIRECTORY etc/ DESTINATION ${CMAKE_INSTALL_ETCDIR} USE_SOURCE_PERMISSIONS
+  install(DIRECTORY etc/ DESTINATION ${CMAKE_INSTALL_SYSCONFDIR} USE_SOURCE_PERMISSIONS
                          PATTERN ".svn" EXCLUDE 
                          REGEX system.rootrc EXCLUDE 
                          REGEX root.mimes EXCLUDE)
diff --git a/cint/reflex/CMakeLists.txt b/cint/reflex/CMakeLists.txt
index 0cda3a0..eca73e9 100644
--- cint/reflex/CMakeLists.txt
+++ cint/reflex/CMakeLists.txt
@@ -27,7 +27,7 @@ if(CMAKE_PROJECT_NAME STREQUAL ROOT)
   ROOT_EXECUTABLE(genmap genmap/genmap.cxx LIBRARIES Reflex ${CMAKE_DL_LIBS})
   #---install headers--------------------------------------------------------
   #ROOT_INSTALL_HEADERS()
-  install(DIRECTORY inc/Reflex/ DESTINATION ${CMAKE_INSTALL_INCDIR}/Reflex
+  install(DIRECTORY inc/Reflex/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/Reflex
                             PATTERN ".svn" EXCLUDE
                             PATTERN "Makefile.am" EXCLUDE
                             REGEX "LinkDef" EXCLUDE )
diff --git a/cmake/modules/RootBuildOptions.cmake b/cmake/modules/RootBuildOptions.cmake
index 4971024..32e1f0c 100644
--- cmake/modules/RootBuildOptions.cmake
+++ cmake/modules/RootBuildOptions.cmake
@@ -179,6 +179,13 @@ endif()
 #---Add Installation Variables------------------------------------------------------------------
 include(RootInstallDirs)
 
+#get_cmake_property(_variableNames VARIABLES)
+#foreach (_variableName ${_variableNames})
+#  if(_variableName MATCHES "^CMAKE_INSTALL_")
+#    message(STATUS "${_variableName}=${${_variableName}}")
+#  endif()
+#endforeach()
+
 
 
 
diff --git a/cmake/modules/RootConfiguration.cmake b/cmake/modules/RootConfiguration.cmake
index fcc336a..d92eb2e 100644
--- cmake/modules/RootConfiguration.cmake
+++ cmake/modules/RootConfiguration.cmake
@@ -26,12 +26,12 @@ set(explicitlink ${value${explicitlink}})
 
 if(gnuinstall)
   set(prefix ${CMAKE_INSTALL_PREFIX})
-  set(etcdir ${CMAKE_INSTALL_FULL_ETCDIR})
+  set(etcdir ${CMAKE_INSTALL_FULL_SYSCONFDIR})
   set(bindir ${CMAKE_INSTALL_FULL_BINDIR})
   set(libdir ${CMAKE_INSTALL_FULL_LIBDIR})
-  set(incdir ${CMAKE_INSTALL_FULL_INCDIR})
+  set(incdir ${CMAKE_INSTALL_FULL_INCLUDEDIR})
   set(mandir ${CMAKE_INSTALL_FULL_MANDIR})
-  set(plugindir ${CMAKE_INSTALL_FULL_ETCDIR}/plugins)
+  set(plugindir ${CMAKE_INSTALL_FULL_SYSCONFDIR}/plugins)
   set(datadir ${CMAKE_INSTALL_FULL_DATADIR})
   set(elispdir ${CMAKE_INSTALL_FULL_ELISPDIR})
   set(ttffontdir ${CMAKE_INSTALL_FULL_FONTDIR})
@@ -45,12 +45,12 @@ if(gnuinstall)
   set(aclocaldir ${CMAKE_INSTALL_FULL_ACLOCALDIR})
 else()
   set(prefix $(ROOTSYS))
-  set(etcdir ${prefix}/${CMAKE_INSTALL_ETCDIR})
+  set(etcdir ${prefix}/${CMAKE_INSTALL_SYSCONFDIR})
   set(bindir ${prefix}/${CMAKE_INSTALL_BINDIR})
   set(libdir ${prefix}/${CMAKE_INSTALL_LIBDIR})
-  set(incdir ${prefix}/${CMAKE_INSTALL_INCDIR})
+  set(incdir ${prefix}/${CMAKE_INSTALL_INCLUDEDIR})
   set(mandir ${prefix}/${CMAKE_INSTALL_MANDIR})
-  set(plugindir ${prefix}/${CMAKE_INSTALL_ETCDIR}/plugins)
+  set(plugindir ${prefix}/${CMAKE_INSTALL_SYSCONFDIR}/plugins)
   set(datadir ${prefix}/${CMAKE_INSTALL_DATADIR})
   set(elispdir ${prefix}/${CMAKE_INSTALL_ELISPDIR})
   set(ttffontdir ${prefix}/${CMAKE_INSTALL_FONTDIR})
@@ -377,11 +377,11 @@ set(pythonvers ${PYTHON_VERSION})
 include(WriteConfigCint)
 WRITE_CONFIG_CINT( ${CMAKE_CURRENT_BINARY_DIR}/tmp/configcint.h)
 execute_process(COMMAND cmake -E copy_if_different ${CMAKE_CURRENT_BINARY_DIR}/tmp/configcint.h ${HEADER_OUTPUT_PATH}/configcint.h)
-#install(FILES ${HEADER_OUTPUT_PATH}/configcint.h DESTINATION ${CMAKE_INSTALL_INCDIR})
+#install(FILES ${HEADER_OUTPUT_PATH}/configcint.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 #---RConfigure.h---------------------------------------------------------------------------------------------
 configure_file(${PROJECT_SOURCE_DIR}/config/RConfigure.in include/RConfigure.h)
-install(FILES ${CMAKE_BINARY_DIR}/include/RConfigure.h DESTINATION ${CMAKE_INSTALL_INCDIR})
+install(FILES ${CMAKE_BINARY_DIR}/include/RConfigure.h DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 #---Configure and install various files----------------------------------------------------------------------
 execute_Process(COMMAND hostname OUTPUT_VARIABLE BuildNodeInfo OUTPUT_STRIP_TRAILING_WHITESPACE )
@@ -502,11 +502,11 @@ install(FILES ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/genreflex
 
 install(FILES ${CMAKE_BINARY_DIR}/include/RConfigOptions.h
               ${CMAKE_BINARY_DIR}/include/compiledata.h 
-              DESTINATION ${CMAKE_INSTALL_INCDIR})
+              DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
 install(FILES ${CMAKE_BINARY_DIR}/etc/root.mimes 
               ${CMAKE_BINARY_DIR}/etc/system.rootrc
-              DESTINATION ${CMAKE_INSTALL_ETCDIR})
+              DESTINATION ${CMAKE_INSTALL_SYSCONFDIR})
               
 install(FILES ${CMAKE_BINARY_DIR}/root-help.el DESTINATION ${CMAKE_INSTALL_ELISPDIR})
 
diff --git a/cmake/modules/RootInstallDirs.cmake b/cmake/modules/RootInstallDirs.cmake
index cbc181a..95f94fc 100644
--- cmake/modules/RootInstallDirs.cmake
+++ cmake/modules/RootInstallDirs.cmake
@@ -7,13 +7,13 @@
 # where <dir> is one of:
 #  BINDIR           - user executables (bin)
 #  LIBDIR           - object code libraries (lib or lib64 or lib/<multiarch-tuple> on Debian)
-#  INCDIR           - C/C++ header files (include)
-#  ETCDIR           - read-only single-machine data (etc)
+#  INCLUDEDIR       - C/C++ header files (include)
+#  SYSCONFDIR       - read-only single-machine data (etc)
 #  DATAROOTDIR      - read-only architecture-independent data (share/root)
 #  DATADIR          - read-only architecture-independent data (DATAROOTDIR)
 #  MANDIR           - man documentation (DATAROOTDIR/man)
 #  MACRODIR         - ROOT macros (DATAROOTDIR/macros)
-#  CINTINCDIR       - CINT include files (LIBDIR/macros)
+#  CINTINCDIR       - CINT include files (LIBDIR/cint)
 #  ICONDIR          - icons (DATAROOTDIR/icons)
 #  SRCDIR           - sources (DATAROOTDIR/src)
 #  FONTDIR          - fonts (DATAROOTDIR/fonts)
@@ -58,44 +58,48 @@ if(NOT DEFINED CMAKE_INSTALL_LIBDIR)
   endif()
 endif()
 
-if(NOT DEFINED CMAKE_INSTALL_CINTINCDIR)
+if(NOT DEFINED CMAKE_INSTALL_INCLUDEDIR)
   if(gnuinstall)
-    set(CMAKE_INSTALL_CINTINCDIR "lib/root/cint" CACHE PATH "cint includes and libraries libraries (cint)")
+    set(CMAKE_INSTALL_INCLUDEDIR "include/root" CACHE PATH "C header files (include)")
   else()
-    set(CMAKE_INSTALL_CINTINCDIR "cint" CACHE PATH "cint includes and libraries libraries (cint)")
+    set(CMAKE_INSTALL_INCLUDEDIR "include" CACHE PATH "C header files (include)")
   endif()
 endif()
 
-if(NOT DEFINED CMAKE_INSTALL_INCDIR)
+if(NOT DEFINED CMAKE_INSTALL_SYSCONFDIR)
   if(gnuinstall)
-    set(CMAKE_INSTALL_INCDIR "include/root" CACHE PATH "C header files (include)")
+    set(CMAKE_INSTALL_SYSCONFDIR "/etc/root" CACHE PATH "read-only single-machine data (etc)")
   else()
-    set(CMAKE_INSTALL_INCDIR "include" CACHE PATH "C header files (include)")
+    set(CMAKE_INSTALL_SYSCONFDIR "etc" CACHE PATH "read-only single-machine data (etc)")
   endif()
 endif()
 
-if(NOT DEFINED CMAKE_INSTALL_ETCDIR)
+if(NOT DEFINED CMAKE_INSTALL_DATAROOTDIR)
   if(gnuinstall)
-    set(CMAKE_INSTALL_ETCDIR "/etc/root" CACHE PATH "read-only single-machine data (etc)")
+    set(CMAKE_INSTALL_DATAROOTDIR "share" CACHE PATH "root for the data (share/root)")
   else()
-    set(CMAKE_INSTALL_ETCDIR "etc" CACHE PATH "read-only single-machine data (etc)")
+    set(CMAKE_INSTALL_DATAROOTDIR "" CACHE PATH "root for the data ()")
   endif()
 endif()
 
-if(gnuinstall)
-  set(__dataroot__ share/root/)
-  set(__share__ share/)
-endif()
-
 #-----------------------------------------------------------------------------
 # Values whose defaults are relative to DATAROOTDIR.  Store empty values in
 # the cache and store the defaults in local variables if the cache values are
 # not set explicitly.  This auto-updates the defaults as DATAROOTDIR changes.
 
+if(NOT CMAKE_INSTALL_CINTINCDIR)
+  if(gnuinstall)
+    set(CMAKE_INSTALL_CINTINCDIR "" CACHE PATH "cint includes and libraries libraries  (LIBDIR/cint)")
+    set(CMAKE_INSTALL_CINTINCDIR "${CMAKE_INSTALL_LIBDIR}/cint")
+  else()
+    set(CMAKE_INSTALL_CINTINCDIR "cint" CACHE PATH "cint includes and libraries libraries (cint)")
+  endif()
+endif()
+
 if(NOT CMAKE_INSTALL_DATADIR)
   set(CMAKE_INSTALL_DATADIR "" CACHE PATH "read-only architecture-independent data (DATAROOTDIR)")
   if(gnuinstall)
-    set(CMAKE_INSTALL_DATADIR "${__share__}root")
+    set(CMAKE_INSTALL_DATADIR "${CMAKE_INSTALL_DATAROOTDIR}/root")
   else()
     set(CMAKE_INSTALL_DATADIR ".")
   endif()
@@ -103,48 +107,80 @@ endif()
 
 if(NOT CMAKE_INSTALL_MANDIR)
   set(CMAKE_INSTALL_MANDIR "" CACHE PATH "man documentation (DATAROOTDIR/man)")
-  set(CMAKE_INSTALL_MANDIR "${__share__}man")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_MANDIR "${CMAKE_INSTALL_DATAROOTDIR}/man")
+  else()
+    set(CMAKE_INSTALL_MANDIR "man")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_MACRODIR)
-  set(CMAKE_INSTALL_MACRODIR "" CACHE PATH "macros documentation (DATAROOTDIR/macros)")
-  set(CMAKE_INSTALL_MACRODIR "${__dataroot__}macros")
+  set(CMAKE_INSTALL_MACRODIR "" CACHE PATH "macros documentation (DATADIR/macros)")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_MACRODIR "${CMAKE_INSTALL_DATADIR}/macros")
+  else()
+    set(CMAKE_INSTALL_MACRODIR "macros")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_ICONDIR)
-  set(CMAKE_INSTALL_ICONDIR "" CACHE PATH "icons (DATAROOTDIR/icons)")
-  set(CMAKE_INSTALL_ICONDIR "${__dataroot__}icons")
+  set(CMAKE_INSTALL_ICONDIR "" CACHE PATH "icons (DATADIR/icons)")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_ICONDIR "${CMAKE_INSTALL_DATADIR}/icons")
+  else()
+    set(CMAKE_INSTALL_ICONDIR "icons")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_FONTDIR)
-  set(CMAKE_INSTALL_FONTDIR "" CACHE PATH "fonts (DATAROOTDIR/fonts)")
-  set(CMAKE_INSTALL_FONTDIR "${__dataroot__}fonts")
+  set(CMAKE_INSTALL_FONTDIR "" CACHE PATH "fonts (DATADIR/fonts)")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_FONTDIR "${CMAKE_INSTALL_DATADIR}/fonts")
+  else()
+    set(CMAKE_INSTALL_FONTDIR "fonts")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_SRCDIR)
-  set(CMAKE_INSTALL_SRCDIR "" CACHE PATH "sources (DATAROOTDIR/src)")
-  set(CMAKE_INSTALL_SRCDIR "${__dataroot__}src")
+  set(CMAKE_INSTALL_SRCDIR "" CACHE PATH "sources (DATADIR/src)")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_SRCDIR "${CMAKE_INSTALL_DATADIR}/src")
+  else()
+    set(CMAKE_INSTALL_SRCDIR "src")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_ACLOCALDIR)
   set(CMAKE_INSTALL_ACLOCALDIR "" CACHE PATH "locale-dependent data (DATAROOTDIR/aclocal)")
-  set(CMAKE_INSTALL_ACLOCALDIR "${__share__}aclocal")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_ACLOCALDIR "${CMAKE_INSTALL_DATAROOTDIR}/aclocal")
+  else()
+    set(CMAKE_INSTALL_ACLOCALDIR "aclocal")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_CMAKEDIR)
   set(CMAKE_INSTALL_CMAKEDIR "" CACHE PATH "CMake modules (DATAROOTDIR/cmake)")
-  set(CMAKE_INSTALL_CMAKEDIR "${__dataroot__}cmake")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_CMAKEDIR "${CMAKE_INSTALL_DATADIR}/cmake")
+  else()
+    set(CMAKE_INSTALL_CMAKEDIR "cmake")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_ELISPDIR)
   set(CMAKE_INSTALL_ELISPDIR "" CACHE PATH "Lisp files (DATAROOTDIR/emacs/site-lisp)")
-  set(CMAKE_INSTALL_ELISPDIR "${__share__}emacs/site-lisp")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_ELISPDIR "${CMAKE_INSTALL_DATAROOTDIR}/emacs/site-lisp")
+  else()
+    set(CMAKE_INSTALL_ELISPDIR "emacs/site-lisp")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_DOCDIR)
-  set(CMAKE_INSTALL_DOCDIR "" CACHE PATH "documentation root (DATAROOTDIR/doc/PROJECT_NAME)")
+  set(CMAKE_INSTALL_DOCDIR "" CACHE PATH "documentation root (DATAROOTDIR/doc/root)")
   if(gnuinstall)
-    set(CMAKE_INSTALL_DOCDIR "${__share__}doc/root")
+    set(CMAKE_INSTALL_DOCDIR "${CMAKE_INSTALL_DATAROOTDIR}/doc/root")
   else()
     set(CMAKE_INSTALL_DOCDIR ".")
   endif()
@@ -152,12 +188,20 @@ endif()
 
 if(NOT CMAKE_INSTALL_TESTDIR)
   set(CMAKE_INSTALL_TESTDIR "" CACHE PATH "root tests (DOCDIR/test)")
-  set(CMAKE_INSTALL_TESTDIR "${CMAKE_INSTALL_DOCDIR}/test")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_TESTDIR "${CMAKE_INSTALL_DOCDIR}/test")
+  else()
+    set(CMAKE_INSTALL_TESTDIR "test")
+  endif()
 endif()
 
 if(NOT CMAKE_INSTALL_TUTDIR)
   set(CMAKE_INSTALL_TUTDIR "" CACHE PATH "root tutorials (DOCDIR/tutorials)")
-  set(CMAKE_INSTALL_TUTDIR "${CMAKE_INSTALL_DOCDIR}/tutorials")
+  if(gnuinstall)
+    set(CMAKE_INSTALL_TUTDIR "${CMAKE_INSTALL_DOCDIR}/tutorials")
+  else()
+    set(CMAKE_INSTALL_TUTDIR "tutorials")
+  endif()
 endif()
 
 
@@ -166,8 +210,8 @@ endif()
 mark_as_advanced(
   CMAKE_INSTALL_BINDIR
   CMAKE_INSTALL_LIBDIR
-  CMAKE_INSTALL_INCDIR
-  CMAKE_INSTALL_ETCDIR
+  CMAKE_INSTALL_INCLUDEDIR
+  CMAKE_INSTALL_SYSCONFDIR
   CMAKE_INSTALL_MANDIR
   CMAKE_INSTALL_DATAROOTDIR
   CMAKE_INSTALL_DATADIR
@@ -188,8 +232,8 @@ mark_as_advanced(
 #
 foreach(dir BINDIR 
             LIBDIR 
-            INCDIR
-            ETCDIR 
+            INCLUDEDIR
+            SYSCONFDIR 
             MANDIR 
             DATAROOTDIR 
             DATADIR 
diff --git a/cmake/modules/RootNewMacros.cmake b/cmake/modules/RootNewMacros.cmake
index 5bb62a5..9d496e0 100644
--- cmake/modules/RootNewMacros.cmake
+++ cmake/modules/RootNewMacros.cmake
@@ -455,7 +455,7 @@ function(ROOT_INSTALL_HEADERS)
     set(dirs inc/)
   endif()
   foreach(d ${dirs})  
-    install(DIRECTORY ${d} DESTINATION ${CMAKE_INSTALL_INCDIR}
+    install(DIRECTORY ${d} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
                            COMPONENT headers 
                            PATTERN ".svn" EXCLUDE
                            PATTERN "X11" EXCLUDE
diff --git a/cmake/modules/SetUpWindows.cmake b/cmake/modules/SetUpWindows.cmake
index 4b5d396..1cfe4d5 100644
--- cmake/modules/SetUpWindows.cmake
+++ cmake/modules/SetUpWindows.cmake
@@ -31,8 +31,8 @@ endif()
 if(CMAKE_PROJECT_NAME STREQUAL ROOT)
   set(CMAKE_CXX_FLAGS "-nologo -I${CMAKE_SOURCE_DIR}/build/win -FIw32pragma.h -FIsehmap.h ${BLDCXXFLAGS} -EHsc- -W3 -wd4244 -D_WIN32")
   set(CMAKE_C_FLAGS   "-nologo -I${CMAKE_SOURCE_DIR}/build/win -FIw32pragma.h -FIsehmap.h ${BLDCFLAGS} -EHsc- -W3 -D_WIN32")
-  install(FILES ${CMAKE_SOURCE_DIR}/build/win/w32pragma.h  DESTINATION ${CMAKE_INSTALL_INCDIR} COMPONENT headers)
-  install(FILES ${CMAKE_SOURCE_DIR}/build/win/sehmap.h  DESTINATION ${CMAKE_INSTALL_INCDIR} COMPONENT headers)  
+  install(FILES ${CMAKE_SOURCE_DIR}/build/win/w32pragma.h  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT headers)
+  install(FILES ${CMAKE_SOURCE_DIR}/build/win/sehmap.h  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR} COMPONENT headers)  
 else()
   set(CMAKE_CXX_FLAGS "-nologo -FIw32pragma.h -FIsehmap.h ${BLDCXXFLAGS} -EHsc- -W3 -wd4244 -D_WIN32")
   set(CMAKE_C_FLAGS   "-nologo -FIw32pragma.h -FIsehmap.h ${BLDCFLAGS} -EHsc- -W3 -D_WIN32")
diff --git a/core/lzma/CMakeLists.txt b/core/lzma/CMakeLists.txt
index 551e946..5652a65 100644
--- core/lzma/CMakeLists.txt
+++ core/lzma/CMakeLists.txt
@@ -16,5 +16,5 @@ list(APPEND LibCore_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/inc)
 set(LibCore_SRCS ${LibCore_SRCS} PARENT_SCOPE)
 set(LibCore_INCLUDE_DIRS ${LibCore_INCLUDE} PARENT_SCOPE)
 
-install(FILES ${LZMA_headers} DESTINATION ${CMAKE_INSTALL_INCDIR})
+install(FILES ${LZMA_headers} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 
diff --git a/core/zip/CMakeLists.txt b/core/zip/CMakeLists.txt
index 0593770..57dfaf1 100644
--- core/zip/CMakeLists.txt
+++ core/zip/CMakeLists.txt
@@ -62,7 +62,7 @@ endif()
 
 # Define all the header files which should be installed when
 # doing a "make install"
-install(FILES ${ZLIB_HEADERS} DESTINATION ${CMAKE_INSTALL_INCDIR})
+install(FILES ${ZLIB_HEADERS} DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
 set_property(GLOBAL APPEND PROPERTY ROOT_INCLUDE_DIRS ${CMAKE_CURRENT_SOURCE_DIR}/inc)
 
 # Add all source files of this subdirectory to the source files
diff --git a/roofit/histfactory/CMakeLists.txt b/roofit/histfactory/CMakeLists.txt
index ee552f9..0f78aab 100644
--- roofit/histfactory/CMakeLists.txt
+++ roofit/histfactory/CMakeLists.txt
@@ -17,6 +17,6 @@ ROOT_EXECUTABLE(hist2workspace MakeModelAndMeasurements.cxx hist2workspace.cxx
                                          Graf Gpad Minuit Foam RooStats XMLParser)
 
 #ROOT_INSTALL_HEADERS()
-install(DIRECTORY inc/RooStats/HistFactory/ DESTINATION ${CMAKE_INSTALL_INCDIR}/RooStats/HistFactory 
+install(DIRECTORY inc/RooStats/HistFactory/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/RooStats/HistFactory 
                        PATTERN ".svn" EXCLUDE
                        REGEX "LinkDef" EXCLUDE )
diff --git a/roofit/roostats/CMakeLists.txt b/roofit/roostats/CMakeLists.txt
index ded7586..385ea0f 100644
--- roofit/roostats/CMakeLists.txt
+++ roofit/roostats/CMakeLists.txt
@@ -10,7 +10,7 @@ ROOT_LINKER_LIBRARY(RooStats  *.cxx G__RooStats.cxx LIBRARIES Core Cint
                                DEPENDENCIES RooFit RooFitCore Tree RIO Hist Matrix MathCore Minuit Foam Graf Gpad )
 
 #ROOT_INSTALL_HEADERS()
-install(DIRECTORY inc/RooStats/ DESTINATION ${CMAKE_INSTALL_INCDIR}/RooStats
+install(DIRECTORY inc/RooStats/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/RooStats
                        COMPONENT headers
                        PATTERN ".svn" EXCLUDE
                        REGEX "LinkDef" EXCLUDE )
diff --git a/tmva/CMakeLists.txt b/tmva/CMakeLists.txt
index 8183c7e..9a5379a 100644
--- tmva/CMakeLists.txt
+++ tmva/CMakeLists.txt
@@ -59,7 +59,7 @@ ROOT_GENERATE_ROOTMAP(TMVA LINKDEF LinkDef1.h LinkDef2.h LinkDef3.h LinkDef4.h
 ROOT_LINKER_LIBRARY(TMVA *.cxx G__TMVA1.cxx G__TMVA2.cxx G__TMVA3.cxx G__TMVA4.cxx LIBRARIES Core Cint 
                     DEPENDENCIES RIO Hist Tree MLP Minuit XMLIO)
 
-install(DIRECTORY inc/TMVA/ DESTINATION ${CMAKE_INSTALL_INCDIR}/TMVA
+install(DIRECTORY inc/TMVA/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/TMVA
                             PATTERN ".svn" EXCLUDE
                             REGEX "LinkDef" EXCLUDE )
 #if(NOT CMAKE_SOURCE_DIR STREQUAL CMAKE_INSTALL_PREFIX)
