avoid problems with spantag.cpp
--- src/Makefile.am.orig
+++ src/Makefile.am
@@ -4,7 +4,7 @@ bin_PROGRAMS = xchm
 xchm_SOURCES = chmapp.cpp chmfile.cpp chmframe.cpp chmfshandler.cpp \
 	chminputstream.cpp chmfontdialog.cpp chmhtmlnotebook.cpp \
 	chmsearchpanel.cpp chmhtmlwindow.cpp chmfinddialog.cpp \
-	chmindexpanel.cpp chmlistctrl.cpp spantag.cpp hhcparser.cpp
+	chmindexpanel.cpp chmlistctrl.cpp hhcparser.cpp
 
 noinst_HEADERS = chmapp.h chmfile.h chmframe.h chmfshandler.h \
 	chminputstream.h chmfontdialog.h chmhtmlnotebook.h \
--- src/Makefile.in.orig
+++ src/Makefile.in
@@ -61,7 +61,7 @@ am_xchm_OBJECTS = chmapp.$(OBJEXT) chmfile.$(OBJEXT) \
 	chmhtmlnotebook.$(OBJEXT) chmsearchpanel.$(OBJEXT) \
 	chmhtmlwindow.$(OBJEXT) chmfinddialog.$(OBJEXT) \
 	chmindexpanel.$(OBJEXT) chmlistctrl.$(OBJEXT) \
-	spantag.$(OBJEXT) hhcparser.$(OBJEXT)
+	hhcparser.$(OBJEXT)
 xchm_OBJECTS = $(am_xchm_OBJECTS)
 xchm_DEPENDENCIES =
 DEFAULT_INCLUDES = -I. -I$(srcdir) -I$(top_builddir)
@@ -203,7 +203,7 @@ INCLUDES = -I$(top_srcdir)/art
 xchm_SOURCES = chmapp.cpp chmfile.cpp chmframe.cpp chmfshandler.cpp \
 	chminputstream.cpp chmfontdialog.cpp chmhtmlnotebook.cpp \
 	chmsearchpanel.cpp chmhtmlwindow.cpp chmfinddialog.cpp \
-	chmindexpanel.cpp chmlistctrl.cpp spantag.cpp hhcparser.cpp
+	chmindexpanel.cpp chmlistctrl.cpp hhcparser.cpp
 
 noinst_HEADERS = chmapp.h chmfile.h chmframe.h chmfshandler.h \
 	chminputstream.h chmfontdialog.h chmhtmlnotebook.h \
@@ -290,7 +290,6 @@ distclean-compile:
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/chmlistctrl.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/chmsearchpanel.Po@am__quote@
 @AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/hhcparser.Po@am__quote@
-@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/spantag.Po@am__quote@
 
 .cpp.o:
 @am__fastdepCXX_TRUE@	if $(CXXCOMPILE) -MT $@ -MD -MP -MF "$(DEPDIR)/$*.Tpo" -c -o $@ $<; \
