--- source/interfaces/basic/src/G4UIQt.cc	2011-12-19 13:34:32.000000000 +0100
+++ source/interfaces/basic/src/G4UIQt.cc	2011-12-19 15:36:00.000000000 +0100
@@ -140,7 +140,7 @@
 
   QWidget *mainWidget = new QWidget(fMainWindow);
   fMyVSplitter = new QSplitter(Qt::Horizontal,fMainWindow);
-  fToolBox = new QToolBox();
+  fToolBox = new QTabWidget();
 
   // Set layouts
 
@@ -180,10 +180,9 @@
   // the splitter 
   //  fToolBox->addItem(fVisParametersTBWidget,"Vis parameters");
   //  fToolBox->addItem(fViewComponentsTBWidget,"Viewer components");
-  fToolBox->addItem(fHelpTBWidget,"Help");
-  fToolBox->addItem(fCoutTBWidget,"Cout");
-  fToolBox->addItem(fHistoryTBWidget,"History");
-
+  fToolBox->addTab(fCoutTBWidget,"Output");
+  fToolBox->addTab(fHistoryTBWidget,"History");
+  fToolBox->addTab(fHelpTBWidget,"Help");
 
 
   fToolBox->setSizePolicy (QSizePolicy(QSizePolicy::Fixed,QSizePolicy::Fixed));
--- source/interfaces/basic/include/G4UIQt.hh	2011-12-19 13:34:22.000000000 +0100
+++ source/interfaces/basic/include/G4UIQt.hh	2011-12-19 15:36:44.000000000 +0100
@@ -158,7 +158,7 @@
   QLineEdit * fCommandArea;
   QTextEdit *fCoutTBTextArea;
   QTextEdit *fHelpArea;
-  QToolBox* fToolBox;
+  QTabWidget* fToolBox;
   QStringList fG4cout;
   QLineEdit * fCoutFilter;
 
