--- source/general/include/GateUIcmdWithAVector.hh
+++ source/general/include/GateUIcmdWithAVector.hh
@@ -118,14 +118,14 @@ template<typename vContentType>
 G4String GateUIcmdWithAVector<vContentType>::ConvertToString
 (std::vector<vContentType> vec)
 {
-  char st[100];
+  //char st[100];
   //LF
   //std::ostrstream os(st,100);
-  std::ostringstream os(st,100);
+  std::ostringstream os;
   //LF
   for (G4int i=0; i<vec.size(); i++) os << vec[i] << " "; 
   os << '\0';
-  G4String vl = st;
+  G4String vl = os.str();
   G4cout << "GateUIcmdWithAVector::ConvertToString : " << vl << G4endl;
   return vl;
 }
--- source/physics/src/GateBinaryCascade.cc
+++ source/physics/src/GateBinaryCascade.cc
@@ -2278,7 +2278,7 @@ G4Fragment * GateBinaryCascade::FindFragments()
   }
 
   G4int zCaptured = 0;
-  G4LorentzVector CapturedMomentum=0;
+  G4LorentzVector CapturedMomentum(0);
   for(i = theCapturedList.begin(); i != theCapturedList.end(); ++i)
   {
       CapturedMomentum += (*i)->Get4Momentum();
@@ -2392,7 +2392,7 @@ G4LorentzVector GateBinaryCascade::GetFinalNucleusMomentum()
 // return momentum of nucleus for use with precompound model; also keep transformation to
 //   apply to precompoud products.
 
-  G4LorentzVector CapturedMomentum=0;
+  G4LorentzVector CapturedMomentum(0);
   G4KineticTrackVector::iterator i;
 //  G4cout << "GetFinalNucleusMomentum Captured size: " <<theCapturedList.size() << G4endl;
   for(i = theCapturedList.begin(); i != theCapturedList.end(); ++i)
